components/homepage/About.tsx

```diff
@@ -84,9 +84,9 @@ const About = () => {
             <div className="mx-[1.62rem]">
               <Image
                 className="rounded-[1rem]"
-                src="/Headshot1.jpg"
-                width={625}
-                height={328}
+                src="/Headshot.jpg"
+                width={1450}
+                height={1450}
```

Comment:
This is still a photo of you

Reply:
Did you try editing the image to be smaller instead of using photo that large?

Reply:
I swapped it out for an Avatar.

Reply:
Yeah these are reviews of you. "What **they** say about me"